app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
```

Comment:
- [ ] Here is the project requirement 👇🏻Kindly Make you create 5 different greetings, for the table for storing your messages, Actually you can use the seed to create five different greetings. ✍🏻 

![image](https://user-images.githubusercontent.com/79058364/216478580-31fba987-8dbc-4d98-a4d2-c3194ab2609e.png)

Reply:
@BenMukebo okay thanks i used just my api but i will create the seed thanks for the observation